manager/src/main/java/com/redhat/service/smartevents/manager/dao/EnumTypeManagedResourceStatus.java

```diff
@@ -0,0 +1,40 @@
+package com.redhat.service.smartevents.manager.dao;
+
+import java.util.Objects;
+import java.util.Properties;
+
+import org.hibernate.annotations.FilterDef;
+import org.hibernate.type.EnumType;
+import org.hibernate.usertype.UserType;
+
+import com.redhat.service.smartevents.infra.models.dto.ManagedResourceStatus;
+
+/**
+ * Custom Type for ManagedResourceStatus enumeration.
+ * If we define the type of the applicable {@link FilterDef} as the enum class itself Hibernate uses Java
+ * serialization to a PostgreSQL bytea type as the value in Filters. This does not work since we choose to explicitly
+ * store Enums as {@link javax.persistence.EnumType#STRING}. This class therefore represents a custom type usable in Filters.
+ */
+public class EnumTypeManagedResourceStatus extends EnumType<ManagedResourceStatus> {
```

Comment:
Look how less pretty this hack-around is.. but I think it's a worthy alternative.

Without this we need to convert the `ManagedResourceStatus` enum values to `String` to use in the `FilterDef`.